js/core/player.js

```diff
@@ -75,7 +75,7 @@ define(["dao", "globals", "core/finances", "data/injuries", "data/names", "lib/b
             numerator = 0;
             denominator = 0;
             for (i = 0; i < components.length; i++) {
-                numerator += ratings[components[i]] * weights[i];
+                numerator += (ratings[components[i]] * weights[i]) + ratings.fuzz;
```

Comment:
The fuzz should be added before multiplying by the weight.

To avoid potential problems like this, it might be a good idea to make a function like `applyFuzz` that applies fuzz to the ratings, and then use that everywhere fuzz is applied. Currently you see stuff like this a few places in player.js:

    Math.round(helpers.bound(fp.ratings[options.ratings[k]] + pr.fuzz, 0, 100))

The function could be like

    function fuzzRating(rating, fuzz) {
        return Math.round(helpers.bound(rating + fuzz, 0, 100));
    }

and used like:

    fuzzRating(fp.ratings[options.ratings[k]], pr.fuzz)

Then there is less chance for a mistake to appear.